package.json

```diff
@@ -45,8 +44,8 @@
     ]
   },
   "devDependencies": {
-    "@testing-library/dom": "^7.28.1",
-    "@testing-library/user-event": "^12.5.0",
+    "@testing-library/dom": "^7.31.2",
+    "@testing-library/user-event": "^12.8.3",
     "concurrently": "^5.3.0",
     "json-server": "^0.16.3"
   }
```

Comment:
Every change in package.json should be reverted. Even if it is a necessary change, let's save updates to all dependencies to for a different PR. So just unstage these changes from the PR 👍🏻

Reply:
Changes to package.json reverted.